cookbook/showing-authentication-view/README.md

```diff
@@ -0,0 +1,71 @@
+# Showing Authentication View
+
+When your App requires authentication, you should invite the user to login when he tries to access your app features.
+The `SignIn` App provides a signin form ready to be invoked from other `Starcounter` apps thanks to the `Blending` approche.
```

Comment:
Please only put code in apostrophes - variable names, method names, URIs, etc. There is no need to put SignIn, Starcounter or Blending in apostrophes.

Please use a spell checker to fix typos like *approche*, *exemple*, etc.

Few tools are good at this: https://www.grammarly.com/, http://www.hemingwayapp.com/.